### PR TITLE
Prepare Alfresco Java SDK 5.1.0 Release

### PR DESCRIPTION
**Pull request by mmartinadanx**

### CODE REVIEW

**Review by nazarethjim — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -248,12 +248,89 @@ at [alfresco-java-event-api](alfresco-java-event-api).
...
+    @Inject
+    SubscriptionsApi subscriptionsApi;
+
+    Map<String, String> config = new HashMap<>();
+    config.put("broker-id", "my-broker"); // Id of the a broker in alfresco-event-gateway configuration
+    config.put("destination", "topic:sample-topic"); // Name of the topic to which the gateway shall publish the events
+
+    Filter filter = new Filter();
+    filter.setType("event-type");
+
```
Comment (jlosornogil) [inline README.md:298]: To complete the filter configuration you'd need to add something like this:

```
Map<String, String> filterConfig = new HashMap<>();
filterConfig.put("event-types", "org.alfresco.event.node.Created,org.alfresco.event.node.Updated"); // Comma-separated list of event types accepted by the filter

filter.setConfig(filterConfig);
```

**Review by jlosornogil — CHANGES_REQUESTED** [review]
I've added a comment about how to configure the event type filter in the sample.

File: README.md (hunk 1 above)
Comment (mmartinadanx) [reply README.md:298]: Thank you!

**Review by jlosornogil — APPROVED** [review]
LGTM Thanks @mmartinadanx !

**Pull request merged**